src/config/config.ts

```diff
@@ -279,6 +280,20 @@ export class Config extends ConfigFile<ConfigFile.Options, ConfigProperties> {
     });
   }
 
+  /**
+   * Gets default options.
+   *
+   * @param isGlobal Make the config global.
+   * @param filename Override the default file. {@link Config.getFileName}
+   */
+  public static getDefaultOptions(isGlobal = false, filename?: string): ConfigFile.Options {
```

Comment:
I don't think needs to be added. See the note in the `constructor` method

Reply:
Removing